### ISSUE TITLE
BUG: Cannot read property 'name' of undefined on artist page

### ISSUE THREAD

**Issue by SpartakusMd** [Musish/Musish#297]
When you open the page of an artist (Ludovico Einaudi: https://musi.sh/artist/7420827), first it loads some info and in 1 second it becomes blank. When you open the console, you see the error:
```
app.ab3c6a89.js:80 TypeError: Cannot read property 'name' of undefined
    at t.value (app.ab3c6a89.js:56)
    at li (app.ab3c6a89.js:80)
    at ui (app.ab3c6a89.js:80)
    at di (app.ab3c6a89.js:80)
    at Vi (app.ab3c6a89.js:80)
    at Xi (app.ab3c6a89.js:80)
    at ja (app.ab3c6a89.js:80)
    at Ra (app.ab3c6a89.js:80)
    at Oa (app.ab3c6a89.js:80)
    at $i (app.ab3c6a89.js:80)
```

Also, the Network tab displays 404 error:
Request URL: https://musi.sh/artist/7420827
Request Method: GET
Status Code: 404 

**To Reproduce**
1. Go to https://musi.sh/artist/7420827
2. You get a white page

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Chrome 72 & Firefox 65

**Comment by BrychanOdlum** [comment]
Reproduced on Safari 12.x

**Comment by BrychanOdlum** [comment]
Looks to be the result of a change made by Apple to MusicKit.js.

When using:
`await music.api.artist(178834, { include: 'albums' })`

We're no longer sent the artist information, as we are when using:
`await MusicKit.getInstance().api.artist(178834)`

We're only given the albums now, we'll need to make two calls (unless anyone is able to come up with another solution).

I'm not able to work on this right now, maybe this evening. Will accept a PR if anyone is able to fix sooner.

**Comment by BrychanOdlum** [comment]
Should be resolved with #298, let me know if you're still facing issues!
